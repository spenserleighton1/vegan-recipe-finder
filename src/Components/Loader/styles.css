.loader {
  width: 100vw;
  height: 100vh;
  position: fixed;
  top: 45%;
}

.image {
  height: 80px
}

.image {
    -webkit-animation: spin 4000ms infinite linear;
    animation: spin 4000ms infinite linear;
}
@-webkit-keyframes spin {
    0% {
        -webkit-transform: rotate(0deg);
        transform: rotate(0deg);
    }
    100% {
        -webkit-transform: rotate(359deg);
        transform: rotate(359deg);
    }
}
@keyframes spin {
    0% {
        -webkit-transform: rotate(0deg);
        transform: rotate(0deg);
    }
    100% {
        -webkit-transform: rotate(359deg);
        transform: rotate(359deg);
    }
}