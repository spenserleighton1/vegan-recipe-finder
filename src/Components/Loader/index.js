import React from 'react';
import leaf from '../../Images/leaf.svg';
import './styles.css';

const Loader = () => (
  <div className='loader'>
    <img src={ leaf } className="image" alt="laoding" />
  </div>
);

export default Loader;