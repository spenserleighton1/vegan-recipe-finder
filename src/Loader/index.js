import React from 'react';
import carrot from '../Images/carrotspinner.png'
import './styles.css';

const Loader = () => (
<div className='loader'>
  <img src={carrot} className="image" alt="laoding" />
</div>
)

export default Loader;