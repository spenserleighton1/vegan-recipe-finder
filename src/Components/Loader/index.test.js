import React from 'react';
import Loader from './';
import { shallow } from 'enzyme';

describe('Loader', () => {
  it('should match the snapshot', () => {
    const wrapper = shallow(<Loader />)

    expect(wrapper).toMatchSnapshot()
  })
})