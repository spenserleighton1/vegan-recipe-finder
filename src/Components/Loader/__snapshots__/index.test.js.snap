// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Loader should match the snapshot 1`] = `
ShallowWrapper {
  "length": 1,
  Symbol(enzyme.__root__): [Circular],
  Symbol(enzyme.__unrendered__): <Loader />,
  Symbol(enzyme.__renderer__): Object {
    "batchedUpdates": [Function],
    "getNode": [Function],
    "render": [Function],
    "simulateEvent": [Function],
    "unmount": [Function],
  },
  Symbol(enzyme.__node__): Object {
    "instance": null,
    "key": undefined,
    "nodeType": "host",
    "props": Object {
      "children": <img
        alt="laoding"
        className="image"
        src="leaf.svg"
/>,
      "className": "loader",
    },
    "ref": null,
    "rendered": Object {
      "instance": null,
      "key": undefined,
      "nodeType": "host",
      "props": Object {
        "alt": "laoding",
        "className": "image",
        "src": "leaf.svg",
      },
      "ref": null,
      "rendered": null,
      "type": "img",
    },
    "type": "div",
  },
  Symbol(enzyme.__nodes__): Array [
    Object {
      "instance": null,
      "key": undefined,
      "nodeType": "host",
      "props": Object {
        "children": <img
          alt="laoding"
          className="image"
          src="leaf.svg"
/>,
        "className": "loader",
      },
      "ref": null,
      "rendered": Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "alt": "laoding",
          "className": "image",
          "src": "leaf.svg",
        },
        "ref": null,
        "rendered": null,
        "type": "img",
      },
      "type": "div",
    },
  ],
  Symbol(enzyme.__options__): Object {
    "adapter": ReactSixteenAdapter {
      "options": Object {
        "enableComponentDidUpdateOnSetState": true,
      },
    },
  },
}
`;
